test/test-errors.js

```diff
@@ -2,10 +2,12 @@
 
```

Comment:
I apologize about the large diff. I couldn't stand the `onlyResponse` convention that I introduced.